schemas/com.snowplowanalytics.mobile/deep_link/jsonschema/1-0-0

```diff
@@ -0,0 +1,25 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Entity that indicates a deep-link has been received and processed.",
+	"self": {
+		"vendor": "com.snowplowanalytics.mobile",
+		"name": "deep_link",
+		"format": "jsonschema",
+		"version": "1-0-0"
+	},
+	"type": "object",
+	"properties": {
+		"url": {
+			"type": "string",
```

Comment:
Just in case - there's `uri` format in JSON Schema https://json-schema.org/understanding-json-schema/reference/string.html#resource-identifiers. We use it extensively in Iglu Central, but I don't remember if Schema DDL (table definition genrators) take any advantage of it, so it's pretty optional.